app/src/main/java/org/simple/clinic/bp/BloodPressureRepository.kt

```diff
@@ -62,51 +58,17 @@ class BloodPressureRepository @Inject constructor(
         .toSingleDefault(bloodPressureMeasurement)
   }
 
-  override fun save(records: List<BloodPressureMeasurement>): Completable {
-    return Completable.fromAction { dao.save(records) }
-  }
-
   fun updateMeasurement(measurement: BloodPressureMeasurement): Completable {
     return encounterRepository.updateBloodPressure(measurement)
   }
 
-  override fun recordsWithSyncStatus(syncStatus: SyncStatus): Single<List<BloodPressureMeasurement>> {
+  //TODO: Remove this method once its usage is replaced in ReportPendingRecordsToAnalytics
```

Comment:
This TODO is added because we are not sure what is the expected count to be sent for analytics once Encounter model UI is added. Whether we should send BP count or Encounter count, this is unclear.
We should handle this when we build the UI.